pkg/controller/rest/verifiable/operation.go

```diff
@@ -240,3 +244,33 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 func (o *Operation) GeneratePresentationByID(rw http.ResponseWriter, req *http.Request) {
 	rest.Execute(o.command.GeneratePresentationByID, rw, req.Body)
 }
+
+// RemoveCredentialByName swagger:route POST /verifiable/credential/remove/name/{name}
+//
+// Removes a verifiable credential by name.
+//
+// Responses:
+//    default: genericError
+//        200: emptyRes
```

Comment:
also, I see that `emptyRes` is not working properly.
the structure should be with the body inside.
e.g 
```
// emptyRes model
//
// swagger:response emptyRes
type emptyRes struct {
Body struct {}
}
```
I also noticed that we are using that structure which means we need to duplicate it.
One structure should be for swagger another structure should be for `Go` usage.

Reply:
the working solution can be like this:
```
type emptyRes struct {}

// emptyResponse model
//
// swagger:response emptyResponse
type emptyResponse struct { // nolint:unused,deadcode
	// in: body
	Body emptyRes
}
```
In that case, you need to replace `200: emptyRes` with `200: emptyResponse`

Reply:
updated to reflect this